build/cmake/utils/CMakeLists.txt

```diff
@@ -39,8 +39,8 @@ if(wxUSE_XRC)
 
         wx_install(CODE "execute_process( \
```

Comment:
```suggestion
        install(CODE "execute_process( \
```

Reply:
It doesn't work with the `wx_install` macro, it resolves the escaped variables I think.